### DISCUSSION TITLE
Is it possible to `redirect()` to an html anchor?

### DISCUSSION THREAD

**Discussion by charnould** [honojs/hono#1326] [Q&A] [answered]
I've a simple controller with a `return c.redirect('/')`. Nothing to say, it works as expected.  

However, I'd like to redirect to a **specific location** in my page (= an **anchor**).  
`return c.redirect('/#embed')` doesn't work, because `#embed` isn't sent.

Is there a way to do it with `Hono`?  
Thanks

**Comment by yusukebe** [comment, marked as answer]
Hi. 

Which runtime do you run your app?

I've tested the following app on Cloudflare/Wrangler and this works well. It is redirected to `http://localhost:8787/foo#bar`.

```ts
app.get('/', (c) => {
  return c.redirect('/foo#bar')
})
```

**Reply by charnould** [reply]
Thanks. It was not `Hono`/`Bun`-related but `Hotwired`-related (see [here](https://github.com/hotwired/turbo/issues/211)).
A simple `data-turbo="false"` works for me.
